core/src/text/fontContext.cpp

```diff
@@ -242,6 +248,73 @@ bool FontContext::layoutText(TextStyle::Parameters& _params, const std::string&
     return true;
 }
 
+void FontContext::fetch(const FontDescription& _ft) {
+
+    const static std::regex regex("^(http|https):/");
+    std::smatch match;
+
+    if (std::regex_search(_ft.uri, match, regex)) {
+        // Load remote
+        resourceLoad++;
+        startUrlRequest(_ft.uri, [&, _ft](std::vector<char>&& rawData) {
+            if (rawData.size() == 0) {
+                LOGE("Bad URL request for font %s at URL %s", _ft.alias.c_str(), _ft.uri.c_str());
+            } else {
+                std::lock_guard<std::mutex> lock(m_mutex);
```

Comment:
There's no runtime cost in `reinterpret_cast`, except if we want to save the `int`. I'd rather leave it at the beginning of the block for easier reading. otherwise I'll be tempted to write another `{`,`}` to guard.